test/common/http/codec_impl_fuzz_test.cc

```diff
@@ -681,6 +708,11 @@ DEFINE_PROTO_FUZZER(const test::common::http::CodecImplFuzzTestCase& input) {
     TestUtility::validate(input);
     codecFuzz(input, HttpVersion::Http1);
     codecFuzz(input, HttpVersion::Http2);
+    // Enable legacy HTTP1 codec.
```

Comment:
This comment seems out of sync with what is done below.  I recommend you explicitly set the runtime flag for the 2 HTTP1 runs, do not rely on the default value of the runtime flag.

Reply:
Oops the comment was incorrect. Line 709 is legacy, this is new. I will explicitly add the runtime vals.